__tests__/app.test.js

```diff
@@ -245,7 +245,55 @@ describe("POST /api/reviews/:review_id/comments", () => {
       .send(testComment)
       .expect(400)
       .then(({ body }) => {
-        expect(body.msg).toBe("Invalid Data Format for ID");
+        expect(body.msg).toBe("Invalid Data Type Used");
+      });
+  });
+});
+
+describe("PATCH /api/reviews/:review_id", () => {
+  it("200: should amend review votes by number provided in incVotes object", () => {
+    const testIncVotes = { inc_votes: 10 };
+
+    const testOriginalReview = {
+      title: "Ultimate Werewolf",
+      designer: "Akihisa Okui",
+      owner: "bainesface",
+      review_img_url:
+        "https://images.pexels.com/photos/5350049/pexels-photo-5350049.jpeg?w=700&h=700",
+      review_body: "We couldn't find the werewolf!",
+      category: "social deduction",
+      created_at: new Date(1610964101251),
+      votes: 5,
+    };
+
+    return request(app)
+      .patch("/api/reviews/3")
+      .send(testIncVotes)
+      .expect(200)
+      .then((response) => {
+        const { review } = response.body;
+        expect(review).toBeInstanceOf(Object);
+        expect(review.votes).toBe(15);
```

Comment:
We need to check you're getting the whole object back with all the keys, with the updated votes as well